Source/Extensions/Blazorise.Charts/ChartOptions.cs

```diff
@@ -330,6 +332,22 @@ public class AxisTicks
         /// </summary>
         [DataMember( EmitDefaultValue = false )]
         public string CallbackJavaScript { get; set; }
```

Comment:
Do we need to have this now with the Callback? Can the Callback be serialized automatically with `Converters.ToDictionary`?

Reply:
1. See my other comment.

2. And maybe it's good to keep the CallbackJavaScript, to give the user total freedom to write JavaScript code.

Reply:
I don't think Callback will prevent them from writing JavaScript code. As far as I can see there is no difference between what `Callback` and `CallbackJavaScript` return. It's just that with `Callback` you also have access to the params of the function, which is good.

Reply:
OK. In that case, I can remove the CallbackJavaScript and call the converter in the Converters.ToDictionary.

(However this means that you cannot use the callback when using **OptionsAsJson**.)